### PR TITLE
Upgrade black 20.8b1

### PR DESCRIPTION
**Pull request by jnoortheen**
- I think it is time to upgrade black. 
- It can now support new lines with comma
- Docstring formatting is also nice

### CODE REVIEW

**Comment by jnoortheen** [comment]
This PR depends on the CI fix from my other PR. Once that is merged/closed we can start merging this maye after some discussions.

File: requirements/tests.txt [hunk 1]
```diff
@@ -8,6 +8,6 @@ prompt-toolkit>=3.0
 pygments>=2.2
 codecov
 coverage
-black==19.10b0 --pre
+black==20.8b1 --pre
```
Comment (scopatz) [inline requirements/tests.txt:11]: Can this be `>=20.8b1`

**Comment by scopatz** [comment]
Great! It looks like this doesn't totally break PLY anymore.

**Comment by scopatz** [comment]
Thanks @jnoortheen!

File: requirements/tests.txt (hunk 1 above)
Comment (jnoortheen) [reply requirements/tests.txt:11]: For the same reason as previous PRs. The future release may break the code.

**Pull request merged**